games/c.yaml

```
@@ -230,6 +230,7 @@
   - TypeScript
   license:
   - MIT
+content: commercial
```

```suggestion
  content: commercial
```
oops; I think the indentation is wrong

---

It should be ok in PR itself, I fixed it